Source/UI/SecurityCodeComponent/SecurityCodeComponent.swift

```diff
@@ -0,0 +1,61 @@
+// 
+//  SecurityCodeComponent.swift
+//  
+//
+//  Created by Okhan Okbay on 26/09/2023.
+//
+
+import Checkout
+import UIKit
+
+public final class SecurityCodeComponent: UIView {
+  private var view: SecurityCodeView!
+
+  private var configuration: SecurityCodeComponentConfiguration!
+  private var isSecurityCodeValid: ((Bool) -> Void)!
+  private var cardValidator: CardValidating!
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+  }
+}
+
+extension SecurityCodeComponent {
+  public func configure(with configuration: SecurityCodeComponentConfiguration,
```

Comment:
Do you want to add any comments for public methods? which can be useful for docs